spec/system/donation_system_spec.rb

```diff
@@ -559,6 +559,22 @@
       it "does not allow deletion of a donation" do
         expect(page).to_not have_link("Delete")
       end
+
+      it "displays donation comment" do
+        expect(page).to have_css("#donation-notes")
+        within "#donation-notes" do
+          expect(page).to have_content("It's a fine day for diapers.")
+        end
+      end
+
+      it "displays default message if no comment" do
+        donation = create(:donation, :with_items, comment: nil)
+
+        visit @url_prefix + "/donations/#{donation.id}"
+        within "#donation-notes" do
+          expect(page).to have_content("None provided")
+        end
+      end
```

Comment:
```suggestion
      context 'when there is no comment defined' do
         before do
                 donation = create(:donation, :with_items, comment: nil)
                 visit @url_prefix + "/donations/#{donation.id}"
         end 
         
         it 'displays the None provided as the comment ' do
            ...
         end 
      end
```
I'd put the "conditions" or "context" above and then the "it" to talk about what is expected. The `context` used in this way avoids cases in which you'd have long-running `it` statements in case the conditionals get more advance or complex.

Let me know if you have questions about this.

Reply:
@edwinthinks I made those changes, let me know what you think!